dive_core/common/emulate_pm4.cpp

```diff
@@ -762,21 +768,26 @@ bool EmulatePM4::AdvanceCb(const IMemoryManager &mem_manager,
             return false;
         }
 
-        uint64_t ib_addr = ((uint64_t)packet.bitfields1.ADDR_HI << 32) |
-                           (uint64_t)packet.bitfields0.ADDR_LO;
-        emu_state_ptr->GetCurIb()->m_ib_queue_index = 0;
-        emu_state_ptr->GetCurIb()->m_ib_queue_size = 0;
-        if (!QueueIB(ib_addr,
-                     packet.bitfields2.DWORDS,
-                     false,
-                     IbType::kContextSwitchIb,
-                     emu_state_ptr))
+        // Sometimes this packet is used for purposes other than to jump to an IB. Check size.
+        // Example: When TYPE is SAVE_IB
```

Comment:
out of curiosity, what does this `save_ib` do?

Reply:
I have no idea, tbh. I just noticed in the captures I'm looking at that these packets with SAVE_IB type have no IBs.